ui/desktop/app/templates/scopes/scope/-header-nav.hbs

```diff
@@ -1,26 +0,0 @@
-{{!
-  Copyright (c) HashiCorp, Inc.
-  SPDX-License-Identifier: BUSL-1.1
-}}
-
-<Rose::Dropdown
```

Comment:
Removed this file since I couldn't find any references to it and it doesn't seem to be in use.